.github/workflows/cron_update_modules/script.php

```diff
@@ -0,0 +1,129 @@
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+declare(strict_types=1);
+
+const VENDOR_NAME = 'prestashop/';
+
+function fetchVersionComposerLock(): array {
+    $composerLock = file_get_contents('composer.lock');
+    $composerLock = json_decode($composerLock, true);
+
+    $versions = [];
+    foreach ($composerLock['packages'] as $package) {
+        if (strpos($package['name'], VENDOR_NAME) !== 0) {
+            continue;
+        }
+        $versions[$package['name']] = $package['version'];
+    }
+
+    return $versions;
+}
+
+// Fetch modules from the repository PrestaShop/prestashop-modules
+$hCurl = curl_init();
+curl_setopt($hCurl, CURLOPT_USERAGENT,'PrestaShop/CronUpdateModules');
+curl_setopt($hCurl, CURLOPT_URL, 'https://api.github.com/repos/PrestaShop/prestashop-modules/git/trees/master');
+curl_setopt($hCurl, CURLOPT_RETURNTRANSFER, 1);
+$gitJSON = curl_exec($hCurl);
+curl_close($hCurl);
+
+if (empty($gitJSON)) {
+    echo 'Empty return from the API';
+    die();
+}
+$gitJSON = json_decode($gitJSON, true);
+if (empty($gitJSON['tree'])) {
+    echo 'Empty return from the API';
+    die();
+}
+$modulesGit = array_filter($gitJSON['tree'], function(array $val) {
+    return $val['type'] === 'commit';
+});
+
+// Fetch packages from the composer.json
+$composerJSON = file_get_contents('composer.json');
+if (empty($gitJSON)) {
+    echo 'Empty return from composer.json';
+    die();
+}
+$composerJSON = json_decode($composerJSON, true);
+if (empty($composerJSON['require'])) {
+    echo 'No required packages in composer.json';
+    die();
+}
+$modulesComposer = array_keys($composerJSON['require']);
+
+// Fetch versions from the composer.lock
+$moduleComposerLockOriginal = fetchVersionComposerLock();
+
+// Update modules with Composer
+foreach ($modulesGit as $moduleGit) {
+    if (!in_array(VENDOR_NAME . $moduleGit['path'], $modulesComposer)) {
+        continue;
+    }
+    echo PHP_EOL . ' === Module prestashop/' . $moduleGit['path'] . PHP_EOL;
+    exec(sprintf(
+        'composer update prestashop/%s',
+        $moduleGit['path']
+    ));
+}
+
+// Fetch versions from the composer.lock
+$moduleComposerLockAfterUpdate = fetchVersionComposerLock();
+
+// Versions
+$pullRequestBodyBumpModules = '';
+foreach ($moduleComposerLockOriginal as $moduleName => $moduleVersion) {
+    if (!array_key_exists($moduleName, $moduleComposerLockAfterUpdate)) {
+        continue;
+    }
+    if ($moduleVersion === $moduleComposerLockAfterUpdate[$moduleName]) {
+        continue;
+    }
+    $pullRequestBodyBumpModules .= 'Module `' . $moduleName. '` : Bump version from `' . $moduleVersion
+        . '` to `' . $moduleComposerLockAfterUpdate[$moduleName] . '`<br />';
+}
+
+if (empty($pullRequestBodyBumpModules)) {
+    echo '';
+    exit();
+}
+
+$pullRequestBody =
+    '| Questions         | Answers'
```

Comment:
```suggestion
    '| Questions         | Answers' . PHP_EOL
```

Otherwise, it will be a one-line string.

Reply:
Fixed